### ISSUE TITLE
Support for popular life support mods

### ISSUE THREAD

**Issue by zorg2044** [zorg2044/BoringCrewServices#10]
- [x] TAC Life support
- [x] Kerbalism (definitely need help from someone or perhaps better yet should be supported from kerbalisms side if its under active development)
- [x] Snacks!

**Comment by EmberSkyMedia** [comment]
I'm assuming enough for 5 kerbals to Minmus and back + 3 days buffer would be enough?

**Comment by OrbitalManeuvers** [comment]
Snacks!'s automatic patching puts 250 snacks in the command module, which is about 16 days for 5.

**Comment by EmberSkyMedia** [comment]
> Snacks!'s automatic patching puts 250 snacks in the command module, which is about 16 days for 5 (default snacks settings)
That's plenty for a Minmus Mission!

**Comment by EmberSkyMedia** [comment]
TACLS (Untested)
@PART[BCS_Centauri_CrewCapsule]:NEEDS[TacLifeSupport]:FOR[BoringCrewService]
{
    RESOURCE
    {
        name = Food
        amount = 20.1
        maxAmount = 20.1
        @amount *= #$/CrewCapacity$
        @maxAmount *= #$/CrewCapacity$
    }
    RESOURCE
    {
        name = Water
        amount = 13.3
        maxAmount = 13.3
        @amount *= #$/CrewCapacity$
        @maxAmount *= #$/CrewCapacity$
    }
    RESOURCE
    {
        name = Oxygen
        amount = 2034
        maxAmount = 2034
        @amount *= #$/CrewCapacity$
        @maxAmount *= #$/CrewCapacity$
    }
    RESOURCE
    {
        name = CarbonDioxide
        amount = 0
        maxAmount = 1757
        @maxAmount *= #$/CrewCapacity$
    }
    RESOURCE
    {
        name = Waste
        amount = 0
        maxAmount = 17
        @maxAmount *= #$/CrewCapacity$
    }
    RESOURCE
    {
        name = WasteWater
        amount = 0
        maxAmount = 1.827
        @maxAmount *= #$/CrewCapacity$
    }
}

**Comment by zorg2044** [comment]
94808cdb438b7403fad740a0a8452af65995b40d

**Comment by EmberSkyMedia** [comment]
Okay I tested Kerbalism, with default settings.

I attempted 2 Mun missions and there is just enough resources for a 5 crew to the Mun and back Fly-By/Orbit/Return, however in both cases the solar panels failed on the return and without repair kits (and an engineer) your dead in the water in hours.

So the capsule is "good enough" as far as I'm concerned, but don't forget to pack plenty of repair kits, it takes 4 for each solar panel on the Service Module to be repaired.
